Corona-Warn-App/src/main/AndroidManifest.xml

```diff
@@ -81,6 +81,7 @@
             android:exported="false"
             android:screenOrientation="portrait"
             android:launchMode="singleTop"
+            android:label=""
```

Comment:
Would be good to add a comment on why it is set to an empty string. Maybe use a string resource with a descriptive name.

Reply:
Done 👍